exec/vector/src/main/codegen/templates/VariableLengthVectors.java

```diff
@@ -253,7 +253,7 @@ public int getPayloadByteCount() {
       // If 1 or more values, then the last value is set to
       // the offset of the next value, which is the same as
       // the length of existing values.
-      return a.get(count-1);
+      return a.get(count-1) + offsetVector.getPayloadByteCount();
```

Comment:
Added a comment. Basically, for planning purposes, the "payload" bytes for a Varchar vector include the bytes taken to hold the text data PLUS the bytes needed by the offset vector that points to those bytes. When averaged, it means a Varchar column width is 4 + avg. byte width.